go/ql/test/library-tests/semmle/go/dataflow/PromotedMethods/DataFlowConfig.ql

```diff
@@ -1,4 +1,5 @@
 import go
+import TestUtilities.InlineExpectationsTest
```

Comment:
Why is this import needed?

Reply:
This was originally exposed through `InlineFlowTest`, which it is no longer. It's needed later in this file, because it also defines an inline expectation test next to an inline flow test.

Reply:
I'd happily expose this again through `InlineFlowTest` (and the same below), but this seems somewhat cleaner.

Reply:
Yeah, this is certainly better.